.env.babylon-integration.example

```diff
@@ -64,4 +64,20 @@ STAKING_TIME=10000 # ~70 days
 STAKING_AMOUNT=10000 # 0.0001 BTC
 
 # Finality Explorer Configuration
-NEXT_PUBLIC_FINALITY_GADGET_API_URL=http://11.22.33.44:18080
\ No newline at end of file
+NEXT_PUBLIC_FINALITY_GADGET_API_URL=http://11.22.33.44:18080
+
+# DNS Configuration
+FINALITY_SYSTEM_SERVER_IP=11.22.33.44
```

Comment:
do we need to add another PR to devops to set these env vars?

Reply:
btw, since we clone this example file to create the .env, it may be better to leave these values unset so as to not break the existing script. if deployment doesnt need SSL, it should work without setting these values

Reply:
yes we could add it to devops

how will these value break the existing script if they are not used in those scripts?

if the deployment doesn't need SSL, these vars won't be used?

Reply:
ah got it. let me remove <> then

Reply:
pushed the fix to main